src/App/Application.cpp

```diff
@@ -2483,35 +2459,35 @@ void processProgramOptions(const variables_map& vm, std::map<std::string,std::st
         mConfig["OpenFileCount"] = buffer.str();
     }
 
-    if (vm.count("output")) {
-        string file = vm["output"].as<string>();
+    if (vm.count("output") != 0U) {
+        auto  file = vm["output"].as<string>();
         mConfig["SaveFile"] = file;
     }
 
-    if (vm.count("hidden")) {
+    if (vm.count("hidden") != 0U) {
         mConfig["StartHidden"] = "1";
     }
 
-    if (vm.count("write-log")) {
+    if (vm.count("write-log") != 0U) {
         mConfig["LoggingFile"] = "1";
         mConfig["LoggingFileName"] = mConfig["UserAppData"] + mConfig["ExeName"] + ".log";
     }
 
-    if (vm.count("log-file")) {
+    if (vm.count("log-file") != 0U) {
         mConfig["LoggingFile"] = "1";
         mConfig["LoggingFileName"] = vm["log-file"].as<string>();
     }
 
-    if (vm.count("user-cfg")) {
+    if (vm.count("user-cfg") != 0U) {
         mConfig["UserParameter"] = vm["user-cfg"].as<string>();
     }

```

Comment:
I don't think I like these ` != 0U` checks. I'm not sure it makes the code easier to read.
I'm open for discussion.

Reply:
I agree that it is good to be explicit with intent, but the intent here is to check if vm _contains_ any of these strings.
Since `.contains(...)` is introduced in c++20 and we're still using c++17,  we're using the next best thing.

C++20 is however just around the corner (about 5 clicks away). It would be best to change these into `.contains(...)` when that is done.